### ISSUE TITLE
Features 'forced' rules are not forced in json output

### ISSUE THREAD

**Issue by markharding** [growthbook/growthbook#222]
'Force Rule' should have `forced: true` boolean in the json output, but it is false.

API returns:
```json
{
  "status": 200,
  "features": {
    "discovery-homepage": {
      "defaultValue": false,
      "rules": [
        {
          "condition": {
            "loggedIn": false
          },
          "variations": [
            false,
            true
          ],
          "weights": [
            0.5,
            0.5
          ],
          "hashAttribute": "id"
        }
      ]
    },
    "channel-gallery": {
      "defaultValue": true,
      "rules": [
        {
          "condition": {
            "id": "100000000000000063"
          },
          "force": false
        },
        {
          "condition": {
            "loggedIn": true
          },
          "force": false
        },
        {
          "variations": [
            true,
            false
          ],
          "weights": [
            0.5,
            0.5
          ],
          "hashAttribute": "id"
        }
      ]
    }
  }
}
```

Screenshot:
<img width="1177" alt="Screenshot 2022-01-20 at 12 54 38" src="https://user-images.githubusercontent.com/851436/150342720-ade4b023-b3e0-459a-b6f6-cfc229666f49.png">

Build: e590310 (2022-01-19)

**Comment by jdorn** [comment]
`"force": false` is actually saying to force the value of the feature to be `false`. I can see how that's confusing though.

**Comment by markharding** [comment]
Ah ok.

Perhaps there is something wrong with the JS SDK though, because its still selecting the 'experiment' and not the first two forced conditions.

**Comment by markharding** [comment]
🎉

**Comment by jdorn** [comment]
Thanks for catching this.  We were missing a unit test case.  Fixed in `@growthbook/growthbook` version 0.16.1
